src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,122 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int MAX_CAPACITY = Integer.MAX_VALUE;
+    private Node<K, V>[] table = new Node[INITIAL_CAPACITY];
+    private int size;
+    private int threshold;
+    private int capacity;
```

Comment:
capacity in unnecessary param that may be replaced with `table.length`

Reply:
I need this variable because table.length is changed only in put. but if we want to count the position of element in resize in the newTable we can't use table.length because we need to use newTable.length. I would delete it, but you told, that I don't need parametr capacity in getIndex() method.